src/activities/photo_hunter/Photo_hunter.qml

```diff
@@ -0,0 +1,176 @@
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.1
+import GCompris 1.0
+
+import "../../core"
+import "photo_hunter.js" as Activity
+
+ActivityBase {
+    id: activity
+
+    onStart: focus = true
+    onStop: {}
+
+    //TODO1: work on display:
+    //-corners of images faded?
+    //-...?
+    //TODO2: animations
+
+    pageComponent: Rectangle {
+        id: background
+        anchors.fill: parent
+        color: "white"
+        signal start
+        signal stop
+
+        Component.onCompleted: {
+            activity.start.connect(start)
+            activity.stop.connect(stop)
+        }
+
+        // Add here the QML items you need to access in javascript
+        QtObject {
+            id: items
+            property Item main: activity.main
+            property alias background: background
+            property alias bar: bar
+            property alias bonus: bonus
+            property var model
+            property alias img1: img1
+            property alias img2: img2
+            property int total
+            property int totalFound: img1.good + img2.good
+
+            property int barHeightAddon: ApplicationSettings.isBarHidden ? 1 : 3
+            property int cellSize: Math.min(background.width / 11 , background.height / (9 + barHeightAddon))
+        }
+
+        onStart: { Activity.start(items) }
+        onStop: { Activity.stop() }
+
+        property bool vert: background.width < background.height
+        property double barHeight: (items.barHeightAddon == 1) ? 0 : 160
+
+        function checkAnswer() {
+            if (items.img1.good + items.img2.good === items.model.length) {
+                bonus.good("flower")
+
+                //remove the problem from the board after first level
+                if (problem!=null) {
+                    frame.anchors.top = background.top
+                    problem.destroy()
+                }
+            }
+        }
+
+        Rectangle {
+            id: problem
+            width: parent.width
+            height: problemTxt.height
+            anchors.top: parent.top
+            anchors.topMargin: 10
+            border.width: 2
+            border.color: "black"
+
+            color: "red"
+
+            GCText {
+                id: problemTxt
+                anchors.centerIn: parent
+                fontSize: items.cellSize / 7.5
+                text: qsTr("Click on the differences between the two images!   " + items.totalFound + "/" + items.total + " found.")
```

Comment:
use qsTr("text %1").arg(variable) when you have variables. Also, avoid having lines bigger than 80 characters if possible